doc/bibliography.rst

```diff
@@ -1,11 +1,11 @@
 Bibliography
 =============
 
-The articles listed here are the papers which cited pyxem.  They might be useful places to start
-if you are interested in the capabilities of pyxem.
+Articles listed here have cited pyxem. They might be useful places to start if you are
+interested in the capabilities of pyxem.
 
-If there are any additional papers you think which should be listed here please contact
-pyxem.team@gmail.com.
+If there are any additional papers you think should be added to the list, please contact
+us at pyxem.team@gmail.com.
 
 .. bibliography:: bibliography.bib
-    :all:
\ No newline at end of file
+    :all:
```

Comment:
Just looking through this and realized that this never got changed.  This is actually the list of papers cited in pyxem not which cite pyxem.

I have a separate list of papers which cite pyxem which we can add.  I know that the reason that I changed this was that people were concerned that the list wouldn't be updated.

Reply:
In the citing pyxem it appears that we just have a link to google scholar which pulls up everything that cites pyxem. That is probably the better thing to do.

Reply:
Aha, of course, I misinterpreted this list. Will rectify.

Reply:
I removed the entire explanation of the bibliography list. If you think I should leave a comment stating that "These articles are referenced in the pyxem documentation", I can do that.

Reply:
No I think Bibliography is explanation enough! 

Looks good to me